### PR TITLE
Update Japanese Translation (Line 865-908)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @atoato88
/cc @YuikoTakada @t-inu @s-kawamura-w664

### CODE REVIEW

**Comment by yosshy** [comment]
I signed it

File: translations/ja.json [hunk 1]
```diff
@@ -864,55 +864,55 @@
...
+	"error provisioning guest": "ゲストのプロビジョン中にエラー",
+	"error starting tunnel": "トンネル開始中にエラー",
+	"error stopping tunnel": "トンネル停止中にエラー",
+	"error: --output must be 'text', 'yaml' or 'json'": "エラー: --output は 'text'、'yaml'、'json' のいずれかでなければなりません",
+	"error: --output must be 'yaml' or 'json'": "エラー: --output は 'yaml'、'json' のいずれかでなければなりません",
+	"experimental": "実験的",
+	"failed to add node": "ノード追加に失敗しました",
+	"failed to open browser: {{.error}}": "ブラウザー起動に失敗しました: {{.error}}",
+	"failed to save config": "設定保存に失敗しました",
+	"failed to set cloud shell kubelet config options": "クラウドシェル kubelete 設定オプションの設定に失敗しました",
```
Comment (atoato88) [inline translations/ja.json:903]:
```suggestion
	"failed to set cloud shell kubelet config options": "クラウドシェル kubelet 設定オプションの設定に失敗しました",
```
s/kubelete/kubelet/

File: translations/ja.json [hunk 2]
```diff
@@ -864,55 +864,55 @@
...
+	"failed to start node": "ノード開始に失敗しました",
+	"fish completion failed": "fish のコマンド補完に失敗しました",
+	"fish completion.": "fish のコマンド補完です。",
+	"if true, will embed the certs in kubeconfig.": "true の場合、Kubernetes に証明書を埋め込みます。",
```
Comment (atoato88) [inline translations/ja.json:907]:
```suggestion
	"if true, will embed the certs in kubeconfig.": "true の場合、kubeconfig に証明書を埋め込みます。",
```
Original word is "kubeconfig".

**Comment by atoato88** [comment]
2 comments.
Could you check those?

File: translations/ja.json [hunk 3]
```diff
@@ -864,55 +864,55 @@
...
-	"config modifies minikube config files using subcommands like \"minikube config set driver kvm2\"\nConfigurable fields: \\n\\n": "",
-	"config view failed": "設定を表示するのに失敗しました",
-	"containers paused status: {{.paused}}": "",
-	"dashboard service is not running: {{.error}}": "ダッシュボードのサービスが動いていません。 {{.error}}",
+	"addon '{{.name}}' is currently not enabled.\nTo enable this addon run:\nminikube addons enable {{.name}}": "'{{.name}}' アドオンは現在無効になっています。\n有効にするためには、以下のコマンドを実行してください。 \nminikube addons enable {{.name}}",
+	"addon '{{.name}}' is not a valid addon packaged with minikube.\nTo see the list of available addons run:\nminikube addons list": "'{{.name}}' は minikube にパッケージングされた有効なアドオンではありません。\n利用可能なアドオンの一覧を表示するためには、以下のコマンドを実行してください。 \nminikube addons list",
+	"addons modifies minikube addons files using subcommands like \"minikube addons enable dashboard\"": "addons コマンドは「minikube addons enable dashboard」のようなサブコマンドを使用することで、 minikube アドオンファイルを修正します",
```
Comment (t-inu) [inline translations/ja.json:871]:
```suggestion
	"addons modifies minikube addons files using subcommands like \"minikube addons enable dashboard\"": "addons コマンドは「minikube addons enable dashboard」のようなサブコマンドを使用することで、minikube アドオンファイルを修正します",
```
直前が読点のため、スペースは不要です。

File: translations/ja.json [hunk 4]
```diff
@@ -864,55 +864,55 @@
...
+	"auto-pause addon is an alpha feature and still in early development. Please file issues to help us make it better.": "auto-pause アドオンはアルファ機能で、未だに開発の初期段階です。auto-pause アドオン改善の手助けのために、問題は報告してください。",
```
Comment (t-inu) [inline translations/ja.json:872]:
```suggestion
	"auto-pause addon is an alpha feature and still in early development. Please file issues to help us make it better.": "auto-pause アドオンはアルファ機能で、まだ開発の初期段階です。auto-pause アドオン改善の手助けのために、問題は報告してください。",
```
「未だに」は意味が少し強く感じたため、「まだ」にしたらどうでしょうか。

**Comment by yosshy** [comment]
@atoato88 @t-inu thank you for your reviews.

**Comment by atoato88** [comment]
/lgtm
Thank you to quick update.
It's OK as Japanese reveiw.

**Comment by yosshy** [comment]
/assign @spowelljr
It’s ready to merge. Would you merge it?

**Review by spowelljr — APPROVED** [review]

**Comment by yosshy** [comment]
@spowelljr thanks for merging it.

**Pull request merged**